examples/3D_contour.py

```diff
@@ -0,0 +1,165 @@
+"""
+A comprehensive example that shows the whole workflow of
+1) Loading data
+2) Defining the model structure for a joint distribution
+3) Estimating the parameter values of the model ("Fitting")
+4) Computing a 3D environmental contour
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage.measure import marching_cubes
+import pandas as pd
+
+from virocon import (
+    read_ec_benchmark_dataset,
+    GlobalHierarchicalModel,
+    LogNormalDistribution,
+    ExponentiatedWeibullDistribution,
+    DependenceFunction,
+    WidthOfIntervalSlicer,
+    HighestDensityContour,
+    plot_marginal_quantiles,
+    plot_dependence_functions,
+)
+
+
+# Load sea state measurements. 
+
+data = pd.read_csv("datasets/NREL_data_oneyear.csv", sep=";", skipinitialspace=True)
+data.index = pd.to_datetime(data.pop(data.columns[0]), format="%Y-%m-%d-%H")
+
+
+# Define the structure of the joint model that we will use to describe
+# the the environmental data. To define a joint model, we define the
+# univariate parametric distributions and the dependence structure.
+# The dependence structure is defined using parametric functions.
+
+# A 3-parameter power function, which will be used as a dependence function.
+def _power3(x, a, b, c):
+    return a + b * x ** c
+
+# A 3-parameter exponential function, which will be used as a dependence function.
+def _exp3(x, a, b, c):
+    return a + b * np.exp(c * x)
+
+def _alpha3(x, a, b, c, d_of_x):
```

Comment:
For consistency, consider adding a one line comment above the _alpha3 function too.